Gemfile.development_dependencies

```diff
@@ -3,7 +3,9 @@
 gem "shakapacker", "7.0.1"
 gem "bootsnap", require: false
 gem "rails", "~> 7.0", ">= 7.0.1"
-gem "sqlite3"
+
+# sqlite3 1.7.0 ends native gem support for Ruby 2.7
```

Comment:
why is sqllite even needed?

Reply:
Needed for dummy app test

I tried to move it to the gemfile of the dummy app but it didn't work. So I kept it here and just fixed the version.